schemas/BaseElements.xsd

```diff
@@ -4447,7 +4447,16 @@
 						<xs:group ref="SystemInfo"/>
 						<xs:element minOccurs="0" name="GlassType" type="GlassType"/>
 						<xs:element minOccurs="0" name="FrameType" type="WindowFrameType"/>
-						<xs:element minOccurs="0" name="Operable" type="HPXMLBoolean"/>
+						<xs:element minOccurs="0" name="Operable" type="HPXMLBoolean">
+							<xs:annotation>
+								<xs:documentation>DEPRECATED. This will be removed in v4.0. Use FractionOperable element instead.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+						<xs:element minOccurs="0" name="FractionOperable" type="HPXMLBoolean">
+							<xs:annotation>
+								<xs:documentation>If the Window/Skylight element represents a single window/skylight, the value should be 0 or 1, otherwise the fraction of area that is operable.</xs:documentation>
```

Comment:
Update the description to indicate that it's the sum of the areas of the windows that can open, not the openable area. Use better words.